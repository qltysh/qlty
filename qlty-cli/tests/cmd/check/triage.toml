bin.name = "qlty"
args = ["check", "--all", "--no-cache", "--json"]
status.code = 1
