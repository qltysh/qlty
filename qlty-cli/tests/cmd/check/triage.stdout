[
  {
    "tool": "untriaged",
    "ruleKey": "fail",
    "message": "untriaged failed",
    "level": "LEVEL_HIGH",
    "category": "CATEGORY_LINT",
    "mode": "MODE_BLOCK"
  },
  {
    "tool": "triaged",
    "ruleKey": "fail",
    "message": "triaged failed",
    "level": "LEVEL_LOW",
    "category": "CATEGORY_STRUCTURE",
    "mode": "MODE_COMMENT"
  }
]
